### PR TITLE
docs(windows): update example config

### PR DESCRIPTION
**Pull request by opakholis**
# Description

- Update `config_win.example.lua`
- Fix typo workspace > workspace

### CODE REVIEW

File: utils/installer/config_win.example.lua [hunk 1]
```diff
@@ -90,8 +93,18 @@ lvim.builtin.nvimtree.setup.renderer.icons.show.git = nil
...
+  "javascript",
+  "json",
   "lua",
+  "python",
+  "typescript",
+  "tsx",
+  "css",
+  "rust",
+  "java",
+  "yaml",
```
Comment (kylo252) [inline utils/installer/config_win.example.lua:107]:
```suggestion
```

File: utils/installer/config_win.example.lua [hunk 2]
```diff
@@ -90,8 +93,18 @@ lvim.builtin.nvimtree.setup.renderer.icons.show.git = nil
 
 -- if you don't want all the parsers change this to a table of the ones you want
 lvim.builtin.treesitter.ensure_installed = {
+  "bash",
   "c",
+  "javascript",
+  "json",
```
Comment (kylo252) [inline utils/installer/config_win.example.lua:99]:
```suggestion
```

File: utils/installer/config_win.example.lua [hunk 3]
```diff
@@ -90,8 +93,18 @@ lvim.builtin.nvimtree.setup.renderer.icons.show.git = nil
 
 -- if you don't want all the parsers change this to a table of the ones you want
 lvim.builtin.treesitter.ensure_installed = {
+  "bash",
```
Comment (kylo252) [inline utils/installer/config_win.example.lua:96]:
```suggestion
```

File: utils/installer/config_win.example.lua [hunk 4]
```diff
@@ -71,6 +73,7 @@ lvim.keys.normal_mode["<C-s>"] = ":w<cr>"
 --   w = { "<cmd>Trouble workspace_diagnostics<cr>", "Workspace Diagnostics" },
 -- }
 
+-- TODO: User Config for predefined plugins
```
Comment (kylo252) [inline utils/installer/config_win.example.lua:76]:
```suggestion
```

**Review by kylo252 — CHANGES_REQUESTED** [review]
Thanks!

The tree-sitter parsers are specifically skipped since gcc may not be available yet.

**Review by kylo252 — APPROVED** [review]

**Comment by opakholis** [comment]
> The tree-sitter parsers are specifically skipped since gcc may not be available yet.

Oh my bad, I don't know at all about that :see_no_evil:

**Pull request merged**